x-pack/plugins/observability_ai_assistant/README.md

```diff
@@ -66,7 +66,7 @@ The knowledge base is an Elasticsearch index, with an inference processor powere
 
 Both the user and the LLM are able to suggest functions, that are executed on behalf (and with the privileges of) the user. Functions allow both the user and the LLM to include relevant context into the conversation. This context can be text, data, or a visual component, like a timeseries graph. Some of the functions that are available are:
 
-- `recall` and `summarize`: these functions query (with a semantic search) or write to (with a summarisation) the knowledge database. This allows the LLM to create a (partly) user-specific working memory, and access predefined embeddings that help improve its understanding of the Elastic platform.
+- `context` and `summarize`: these functions query (with a semantic search) or write to (with a summarisation) the knowledge database. This allows the LLM to create a (partly) user-specific working memory, and access predefined embeddings that help improve its understanding of the Elastic platform.
```

Comment:
The "knowledge base" term is getting overloaded.

Reply:
Can you clarify what you mean? (This is just a find+replace on content that already exists)

Reply:
Maybe I can ask in a different way: what's the difference between context and knowledge base? Because we use both terms interchangeably. 
I thought the knowledge base was our ELSER datastore but it now also includes the consumer-provided context